src/component-library/Stack/Stack.style.tsx

```diff
@@ -22,8 +22,14 @@ const getSpacing = (spacing: StackProps['spacing']) => {
   return result;
 };
 
-export const StackContainer = styled.div<StackProps>`
+type StackContainerProps = {
+  $spacing: Spacing;
+};
+
+const StackContainer = styled.div<StackContainerProps>`
   > *:not(:last-child) {
```

Comment:
Should we make the Stacks full width? @tomjeatt

Reply:
Yes 👍